docs/updates/migrate-v2.md

```diff
@@ -0,0 +1,68 @@
+# Migrate to v2
+
+v2.0.0 of the UI5 Tooling was released on April 1, 2020. As a major feature, it introduces the easy consumption of SAPUI5 libraries in UI5 projects.
+
+## Breaking changes
+**All UI5 Tooling Modules: Require Node.js >= 10**
+
+Support for older Node.js releases has been dropped.
+
+**UI5 Builder: Make namespace mandatory for application and library projects ([SAP/ui5-builder#430](https://github.com/SAP/ui5-builder/pull/430))**
+
+The UI5 Tooling must be able to determine an application- or library project's namespace. Otherwise an error is thrown.
+
+Ideally the namespace should be defined in the `sap.app/id` field of the [`manifest.json`](https://ui5.sap.com/#/topic/74038a52dcd7404e82b38be6d5fb1458)
+
+In case of libraries, additional fallbacks are in place:
+
+1. The `name` attribute defined in the `.library` file
+2. The path of the `library.js` file
+
+**UI5 Builder: LibraryFormatter: Ignore manifest.json of nested apps ([SAP/ui5-builder#392](https://github.com/SAP/ui5-builder/pull/392))**
+
+If a library contains both, a manifest.json and .library file, they must either be located in the same directory or the manifest.json is ignored. In cases where the manifest.json is located on a higher level or different directory on the same level than a .library file, an exception is thrown.
```

Comment:
```suggestion
If a library contains both a manifest.json and a .library file, they must be located in the same directory. Otherwise the manifest.json is ignored. In cases where the manifest.json is located at a higher level or in a different directory at the same level as the .library file, an exception is thrown.
```